ivy_tests/test_ivy/test_functional/test_extensions/test_core/test_manipulation.py

```diff
@@ -414,3 +414,42 @@ def test_top_k(
         largest=largest,
         out=None,
     )
+
+
+# fliplr
+@handle_cmd_line_args
+@given(
+    dtype_and_m=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("numeric"),
```
Probably we can make it `numeric` instead of just `float` to cover more cases.

Reply:
Thank you for noticing it. I've committed the suggested change.